pkg/codegen/java/gen_program.go

```diff
@@ -528,6 +528,29 @@ func pulumiImport(pkg string, module string, member string) string {
 	return "com.pulumi." + sanitizeImport(pkg) + "." + sanitizeImport(module) + "." + member
 }
 
+func pulumiResourceImport(r *pcl.Resource, pkg string, module string, member string) string {
```

Comment:
Worth a comment to mention how this is different/what its intended use is?

Reply:
Done, thanks!